.github/dependency-review/config.yml

```diff
@@ -2,7 +2,6 @@ allow-licenses:
   - '0BSD'
   - 'Apache-2.0'
   - 'Apache-2.0 AND MIT'
-  - 'BlueOak-1.0.0'
```

Comment:
Let's keep BlueOak-1.0.0 since it is tracked

Reply:
Got it, will be updating soon